contracts/ChainConfig.sol

```diff
@@ -60,6 +60,20 @@ contract ChainConfig is Ownable {
     return true;
   }
 
+  /// @dev this is just for development purposes! must be removed in real product implementation
+  function updateMinimumStakingTokenPercentage(uint8 _minimumStakingTokenPercentage)
```

Comment:
I think this should be passed in the constructor so you specify whether the chain does or doesn't support it.
Might be easier than launching two contracts with two versions of the code.

Reply:
done